templates/agent-conf.d/kafka.yaml.erb

```diff
@@ -6,48 +6,52 @@
 # This sample works only for Kafka >= 0.8.2.
 # If you are running a version older than that, you can refer to agent 5.2.x released
 # sample files, https://raw.githubusercontent.com/DataDog/dd-agent/5.2.1/conf.d/kafka.yaml.example
-
 instances:
-  - host: <%= @host %>
-    port: <%= @port %> # This is the JMX port on which Kafka exposes its metrics (usually 9999)
-<% if @tags.length > 0 -%>
+<% @servers.each do |server| -%>
+  - host: <%= server['host'] %>
+    port: <%= server['port'] %>
+  <%- if !server['tags'].nil? && server['tags'].any? -%>
     tags:
-<%- @tags.each do |tag_name, tag_value| -%>
-      <%= tag_name %>: <%= tag_value %>
-      environment: <%= @environment %>
-      app: <%= app %>
-<% end -%>
+    <%- server['tags'].each do |tag| -%>
+      - <%= tag %>
+      - environment: <%= @environment %>	  
+      - app: <%= app %>
+    <%- end -%>
+  <%- end -%>
+  <%- if !server['user'].nil? -%>
```

Comment:
changed this from `username` (https://github.com/DataDog/puppet-datadog-agent/blob/master/templates/agent-conf.d/kafka.yaml.erb#L19-L21) to `user` based on existing config.